src/engine/platform/pokey.cpp

```diff
@@ -454,11 +454,9 @@ void DivPlatformPOKEY::setFlags(const DivConfig& flags) {
     for (int i=0; i<4; i++) {
       oscBuf[i]->rate=rate/2;
     }
-    if (altASAP) {
-      delete altASAP;
-    }
-    altASAP=new AltASAP::Pokey(chipClock,rate);
+    altASAP.init(chipClock,rate);
   } else {
+    MZPOKEYSND_Init(&pokey);
```

Comment:
No need ;p

Reply:
I thought that symmetry is good :)
Reverting.